src/components/UI/navbar/index.js

```diff
@@ -112,26 +114,13 @@ const Navbar = () => {
                 </Link>
               )}
               {isLoggedIn && (
-                <div className={styles.userGreet}>
-                  <Link href={USER_PROFILE_URL}>
-                    <a>
-                      <div className={styles.userGreetMsg}>
-                        {`Hello ${
-                          isLoggedIn ? `${userData.firstName}` : 'User'
-                        }!`}
-                      </div>
-                      <img
-                        className={styles.userProfilePic}
-                        src={
-                          isLoggedIn
-                            ? `${userData.profilePicture}`
-                            : `${DEFAULT_AVATAR}`
-                        }
-                        alt="Profile pic"
-                      />
-                    </a>
-                  </Link>
-                </div>
+                <Dropdown
+                  isLoggedIn={isLoggedIn}
```

Comment:
When will the value be `false` in the isLoggedIn that you are passing in Dropdown component?
Because what I see is that it is displayed only when isLoggedIn is true.

Also, there is no logical sense in importing these constant values in Navbar component and passing them in Dropdown as props, you should rather import them in Dropdown component itself.